test-basic/documents-data-movement-readAll.js

```diff
@@ -496,23 +497,23 @@ describe('data movement readAll', function() {
         uriStream = new Stream.PassThrough({objectMode: true});
         categoriesUrisList.forEach(uri => uriStream.push(uri));
         uriStream.push(null);
-        const readAllStream = dbWriter.documents.readAll({categories: ['rawContent','permissions']});
+        const readAllStream = dbWriter.documents.readAll(uriStream,{categories: ['rawContent','permissions']});
         readAllStream.on('error', function(err){
             err.toString().should.equal('Error: Categories should not have other option(s) if rawContent is needed.');
             done();
         });
 
-        uriStream.pipe(readAllStream);
+      //  uriStream.pipe(readAllStream);
```

Comment:
If this line isn't needed anymore, it's better to delete it so there's not confusion later on over whether it's needed or not.

Reply:
Done.